cli/command/trust/key_load_test.go

```diff
@@ -109,6 +115,7 @@ var testKeys = map[string][]byte{
 }
 
 func TestLoadKeyFromPath(t *testing.T) {
+	skip.If(t, func() bool { return runtime.GOOS == "windows" })
```

Comment:
Instead of  the `func() bool` here you can use `IfCondition`, or update `gotestyourself` to `1.3.0` so that `If` accepts the bool.

Maybe add a reason why as well?